src/sbt-test/sbt-1.3/testkit/tests/src/test/scala/fix/InputOutputSuite.scala

```diff
@@ -2,15 +2,16 @@ package fix
 
 import scalafix.testkit._
 import scala.util.control.NonFatal
+import org.scalatest.FunSuiteLike
 import org.scalatest.exceptions.TestFailedException
 
 object IntputOutputSuite {
   def main(args: Array[String]): Unit = {
     if (Array("--save-expect").sameElements(args)) {
-      val suite = new SemanticRuleSuite(
-        TestkitProperties.loadFromResources(),
-        isSaveExpect = true
-      ) {
+      val suite = new AbstractSemanticRuleSuite with FunSuiteLike {
+        override val props = TestkitProperties.loadFromResources()
+        override val isSaveExpect = true
```

Comment:
Non-trivial usage of `SemanticRuleSuite` that https://github.com/scalacenter/scalafix/pull/1395/files won't cover. See also https://github.com/scalacenter/scalafix/issues/1147.